h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -1356,6 +1356,70 @@ def show_test_results(test_name, curr_test_val, new_test_val):
         print(pass_string)
         return 0
 
+def equal_H2OTwoDimTable(table1, table2, col_header_list, tolerance=1e-6, check_sign=False):
+    """
+    This method compares two H2OTwoDimTables and verify that their difference is less than value set in tolerance. It
+    is probably an overkill for I have assumed that the order of col_header_list may not be in the same order as
+    the values in the table.cell_values[ind][0].  In addition, I do not assume an order for the names in the
+    table.cell_values[ind][0] either for there is no reason for an order to exist.
+
+    :param table1: H2OTwoDimTable to be compared
+    :param table2: the other H2OTwoDimTable to be compared
+    :param col_header_list: list of strings denote names that we can the comparison to be performed
+    :param tolerance: default to 1e-6
+    :param check_sign: bool, determine if the sign of values are important or not.  For eigenvectors, they are not.
```

Comment:
Better make sign conversion utility as a separate method; and ensure to flip sign on the entire vector at once, not on individual components.
Eg: (1, 2, 3) == (-1, -2, -3) but not (1, -2, 3)